ZenPacks/zenoss/Microsoft/Windows/WinService.py

```diff
@@ -164,10 +164,33 @@ def get_winservices_modes(self):
             if svc.monitored():
                 data[svc.id] = {'modes': svc.get_serviceclass_startmodes(),
                                 'mode': svc.startMode,
-                                'monitor': svc.monitored()
+                                'monitor': svc.monitored(),
+                                'severity': svc.getFailSeverity(),
                                 }
         return data
 
+    def getTemplateDefaultService(self):
+        '''Return datasource for template if it exists'''
+        template = self.getRRDTemplate()
+        if template:
+            return template.datasources._getOb('DefaultService', None)
```

Comment:
Users can also create a new datasource for monitoring services, so it won't always be the 'DefaultService'.  you'll probably need to loop through them as we are in monitored() and call getMonitored to see if the datasource applies.

Or, what might be better is to add a new attribute to WinService to save the datasource name.  you can set it when a datasource enables monitoring.  it can be empty when there is no datasource